src/test/java/core/basesyntax/service/impl/CreateFileServiceImplTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CreateFileService;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CreateFileServiceImplTest {
+    private static final String CREATED_FILE = "src/test/resources/newFile.csv";
+    private CreateFileService createFileService;
+
+    @Before
+    public void setUp() {
+        createFileService = new CreateFileServiceImpl();
+    }
+
+    @Test
+    public void create_File_OK() {
```

Comment:
you have different naming patterns for methods in different classes
let's use this pattern `methodName_condition_ok/notOk`